drivers/watchdog/wdt_nrfx.c

```diff
@@ -150,6 +151,15 @@ static void wdt_event_handler(struct device *dev)
 	}
 }
 
+/*
+ * We're only using node labels here because we have per-instance
+ * configuration options like CONFIG_NRFX_WDT0.
+ *
+ * Those are the only obstacle to moving this to DT_INST APIs.
```

Comment:
We need `idx` to contain the hardware instance index, so that it can be used in `nrfx_wdt_##idx##_irq_handler` and in `NRFX_WDT_INSTANCE(idx)`. And using DT_INST cannot provide this. Options like `CONFIG_NRFX_WDT0` only activate the relevant part of nrfx. IMO they are not direct obstacles to using DT_INST.

Reply:
Removing these comments, thanks.